contracts/ronin/validator/CoinbaseExecution.sol

```diff
@@ -117,9 +117,10 @@ abstract contract CoinbaseExecution is
       _slashIndicatorContract.updateCreditScores(_currentValidators, _lastPeriod);
       _currentValidators = _syncValidatorSet(_newPeriod);
     }
-
     _revampBlockProducers(_newPeriod, _currentValidators);
     emit WrappedUpEpoch(_lastPeriod, _epoch, _periodEnding);
+    _periodOf[_epoch] = _lastPeriod;
```

Comment:
On the very first epoch wrapping, the last period is 0, we no longer need this LoC
```suggestion
```